README.md

```diff
@@ -1,43 +1,27 @@
 # bdbcontrib
 
-A set of utilities for bayesdb.
+A set of utilities and a front end for BayesDB.
 
 ## Installing
 
-The software in bdbcontrib requires:
-
-- [bayeslite >=0.1.2](http://probcomp.csail.mit.edu/bayesdb/)
-- [ipython notebook >=3](http://ipython.org/notebook.html)
-- [markdown2](https://pypi.python.org/pypi/markdown2)
-- [matplotlib](http://matplotlib.org/)
-- [numpy](http://www.numpy.org/)
-- [pandas](http://pandas.pydata.org/)
-- [scikit-learn](http://scikit-learn.org/)
-- [seaborn >=6](http://stanford.edu/~mwaskom/software/seaborn/)
-- [tornado >=4](http://www.tornadoweb.org/en/stable/)
-
-The tests require:
-
-- [mock](https://pypi.python.org/pypi/mock)
-- [pillow](https://python-pillow.github.io/)
-- [pytest](http://pytest.org/)
-
-The documentation requires:
-
-- [numpydoc](https://pypi.python.org/pypi/numpydoc)
-- [sphinx](sphinx-doc.org)
-
-Individual parts of bdbcontrib may have slimmer dependencies, if you
-want to pull them out for more limited purposes.
+Software requirements are detailed in setup.py.
+Please see http://probcomp.csail.mit.edu/bayesdb/install.html for installation.
 
 ## Test
 
-To run the paltry automatic tests:
+Please run local tests before sending a pull request:
 
 ```
 $ ./check.sh
 ```
 
+That does not run the complete test suite, only the smoke tests, but
+is usually good enough. For the full suite:
+
+```
+$ ./check.sh tests shell/tests
```

Comment:
Omit `shell/tests` -- it does not exist here, and if you include it then no tests run but pytest indicates success!